### PR TITLE
Fix Optimism scenarios

### PR DESCRIPTION
**Pull request by kevincheng96**
~~This PR refactors the cross-chain governance scenarios so that a single scenario can be used for all L2s/sidechains. We also move cross-chain governance scenarios into their own file.~~

This PR fixes a lot of different errors that are cropping up for Optimism scenarios.

### CODE REVIEW

**Comment by kevincheng96** [comment]
Fixed up all the immediately fixable scenarios for Optimism. 

The one remaining failure for Optimism is due to our scenario code using a Bedrock-only interface when Optimism mainnet hasn't been upgraded yet. The one remaining failure for Optimism-goerli is due to the fact that we only have a single collateral asset in the deploy script right now, which would fail for any scenarios that try to bump the supply cap for a collateral asset that is not asset 0.

**Review by jflatow — APPROVED** [review]
Great work 💯

File: scenario/utils/index.ts [hunk 1]
```diff
@@ -415,6 +416,65 @@ export async function fastL2GovernanceExecute(
...
+  const signatures: string[] = [];
+  const calldata: string[] = [];
+
+  // Create the chain-specific wrapper around the L2 proposal data
+  switch (bridgeNetwork) {
+    case 'optimism':
+    case 'optimism-goerli': {
+      const sendMessageCalldata = utils.defaultAbiCoder.encode(
+        ['address', 'bytes', 'uint32'],
+        [bridgeReceiver.address, l2ProposalData, 1_000_000] // XXX find a reliable way to estimate the gasLimit
```
Comment (jflatow) [inline scenario/utils/index.ts:435]: 🤔

**Review by scott-silver — APPROVED** [review]
👍

**Pull request merged**